tropicalchancer/netsos-old
# Feature/zupass auth

## Changes
- Implemented ZuPass authentication using Zuzalu participant verification
- Added AuthProtection component to protect benefits route
- Added proper TypeScript types for ZuPass prove request

## Testing
1. Visit /benefits page
2. Click "Verify with ZuPass"
3. Verify Zuzalu participant credential flow works
4. Verify protected content shows after successful auth

## Notes
- Implementation based on Zupoll's auth pattern
- Uses Zuzalu participant group (semaphore/1)
